policy/modules/admin/netutils.fc

```diff
@@ -20,3 +20,14 @@
 /usr/sbin/send_arp	--	gen_context(system_u:object_r:ping_exec_t,s0)
 /usr/sbin/tcpdump	--	gen_context(system_u:object_r:netutils_exec_t,s0)
 /usr/sbin/traceroute.*	--	gen_context(system_u:object_r:traceroute_exec_t,s0)
+
+ifdef(`distro_gentoo',`
+/usr/bin/arping	--	gen_context(system_u:object_r:netutils_exec_t,s0)
+/usr/bin/fping 	--	gen_context(system_u:object_r:ping_exec_t,s0)
+/usr/bin/hping2	--	gen_context(system_u:object_r:ping_exec_t,s0)
+/usr/bin/iftop		--	gen_context(system_u:object_r:netutils_exec_t,s0)
+/usr/bin/iptstate	--	gen_context(system_u:object_r:netutils_exec_t,s0)
+/usr/bin/send_arp	--	gen_context(system_u:object_r:ping_exec_t,s0)
+/usr/bin/tcpdump	--	gen_context(system_u:object_r:netutils_exec_t,s0)
+/usr/bin/traceroute.*	--	gen_context(system_u:object_r:traceroute_exec_t,s0)
+')
```

Comment:
The build errors are coming from here:

```
/tmp/selinux/sbin/setfiles: file_contexts: Multiple same specifications for /usr/bin/traceroute.*.
/tmp/selinux/sbin/setfiles: file_contexts: Multiple same specifications for /usr/bin/fping.
/tmp/selinux/sbin/setfiles: file_contexts: Multiple same specifications for /usr/bin/arping.
/tmp/selinux/sbin/setfiles: file_contexts: Multiple same specifications for /usr/bin/hping2.
/tmp/selinux/sbin/setfiles: file_contexts: Multiple same specifications for /usr/bin/smartd.
/tmp/selinux/sbin/setfiles: file_contexts: Multiple same specifications for /usr/bin/tcpdump.
/tmp/selinux/sbin/setfiles: file_contexts: Multiple same specifications for /usr/bin/iptstate.
/tmp/selinux/sbin/setfiles: file_contexts: Multiple same specifications for /usr/bin/send_arp.
```

Remove the lines causing these errors from your PR here and it will fix them.

Reply:
My bad, fixed.